network/can: properly handle CAN.RestartSec=0

rzblue opened the pull request:
Previously, 0 was ignored and restart-ms wouldn't get added to the netlink message. Instead, set the attribute unconditionally.

yuwata commented on src/network/networkd-can.c:
```diff
@@ -89,7 +89,7 @@ int can_set_netlink_message(Link *link, sd_netlink_message *m) {
         }
 
-        if (link->network->can_restart_us > 0) {
+        if (link->network->can_restart_us_set) {
```
It is not necessary to introduce the flag. Please simply drop the `if` condition, and set the attribute unconditionally.

rzblue replied:
I'd considered that at first, but wasn't sure what the intended semantics were for an unset value (actively push a value to the kernel vs do nothing). I'll go ahead and make that change.

yuwata reviewed (changes requested):
Thank you for working on the issue!
Please break the long line in the commit message.

rzblue committed:
network/can: properly handle CAN.RestartSec=0

Previously, 0 was ignored and restart-ms wouldn't get added to the
netlink message. Instead, set the attribute unconditionally.
